components/drawer/Drawer.razor.cs

```diff
@@ -442,33 +439,16 @@ protected override async Task OnAfterRenderAsync(bool isFirst)
         }
 
         private Timer _timer;
-        private int _zIndex = 1000;
+        private int _zIndex = DefaultZIndez;
         private string _zIndexStyle = "";
 
-        private void TriggerPlacementChangeCycleOnce()
```

Comment:
@zxyao145 Please check if this method is useless.

Reply:
Yes, this method will only be triggered when the Placement changes (line 394).

Reply:
Thanks @zxyao145 , please help to confirm whether there is any problem after deletion.